packages/api/tests/endpoints/rules.js

```diff
@@ -281,6 +350,7 @@ test('POST returns a record exists when one exists', (t) => {
 
 test('PUT updates a rule', (t) => {
   const newRule = Object.assign({}, testRule, { state: 'ENABLED' });
+  console.log(`NewRule: ${JSON.stringify(newRule)}`);
```

Comment:
Did you mean to leave these console statements in?

Reply:
For now, yes. I haven't quite worked out the kinks, yet.